bitnami/odoo/templates/externaldb-secrets.yaml

```diff
@@ -7,7 +7,7 @@ SPDX-License-Identifier: APACHE-2.0
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ printf "%s-externaldb" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
+  name: {{ printf "%s-externaldb" .Release.Name | trunc 63 | trimSuffix "-" }}
```

Comment:
I think the old approach is the right one to follow, please perform the changes in 

https://github.com/bitnami/charts/blob/09b80f82df6a51202b219805ae4b0d743ae406cc/bitnami/odoo/templates/_helpers.tpl#L100-L102

to use `(include "common.names.fullname" .)` instead 😁